README.md

```diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
 * [How to make all-sky plots](https://github.com/desihub/desiutil/blob/master/doc/nb/SkyMapExamples.ipynb)
 * [Working with DESI target bits (Main, CMX, SV)](https://github.com/desihub/desitarget/blob/master/doc/nb/target-selection-bits-and-bitmasks.ipynb)
 
+# External databases
+
+In addition to the database access through NERSC, one can use the following external database services.
+
+## Astro Data Lab
+
+The Astro Data Lab serves a copy of the DESI EDR database as `desi_edr`. This is accessible to users without a NERSC account. 
+Various modes of data access are described [here](https://datalab.noirlab.edu/desi/access.php). 
+In particular, there is a Table Access Protocol (TAP) handle that provides a convenient access layer for the 
+DESI catalog database tables. TAP-aware clients (such as TOPCAT) can point to `https://datalab.noirlab.edu/tap`, 
+select the `desi_edr` database, and see the database tables and descriptions. 
+
+Descriptions of the associated tables can also be found in the Data Lab [table browser](https://datalab.noirlab.edu/query.php?name=desi_edr.zpix) 
+and on the DESI Data Documentation [Database page](https://data.desi.lbl.gov/doc/access/database/). 
+
+## SPARCL
+
+The [SPectra Analysis & Retrievable Catalog Lab (SPARCL)](https://astrosparcl.datalab.noirlab.edu) contains 
+DESI EDR spectra that were coadded per healpix. It is a searchable database that can be used via the Astro Data Lab 
+JupyterLab Notebook. Alternatively, the Python client can be installed locally using:
+```
+pip install sparclclient
+```
+The client can be loaded within a Python session or program via:
+```
+>> from sparcl.client import SparclClient 
```

Comment:
The standard Python prompt has three arrows: `>>>`.

Reply:
fixed.